src/airport.js

```diff
@@ -1 +1,53 @@
+class Airport {
+    #airportCapacity;
+    #planesStanding;
 
+    constructor(airportCapacity = 10, planesStanding = []) {
+        this.#airportCapacity = airportCapacity;
+        this.#planesStanding = planesStanding;
+    }
+
+    landPlane(plane) {
+        if (!plane?.id) return `Unidentified planes are not allowed at the airport`
+        if (this.fullAirport()) {
+            return `Sorry, ${plane.id}, no room for you to park!`;
```

Comment:
maybe my messages are too airport specific, I need to review how to write them in the index file instead of the class to keep things tidy.